src/main/java/com/quizapp/enterprise/services/GameService.java

```diff
@@ -46,7 +49,13 @@ public void joinGame(String gameId, Player playerToJoin) throws Exception {
     }
 
     private boolean userNameExists(String userName, String gameCode){
-        return GameTracker.getInstance().getGameByCode(gameCode).getPlayers().stream().anyMatch(player -> player.getPlayerUsername().equals(userName));
+        return GameTracker
+                .getInstance()
+                .getGameByCode(gameCode)
+                .getPlayers()
+                .stream()
+                .anyMatch(player -> player.getPlayerUsername()
+                        .equals(userName));
```

Comment:
reformatted the streams to use indentation so the code may be cleaner and easier to read.
This recommended by the "Iterators and Streams" section of the java best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3